### DISCUSSION TITLE
[SOLVED] How to get full text from heise.de ?

### DISCUSSION THREAD

**Discussion by user858753257** [martinrotter/rssguard#557] [General]
Hi , what should I do to see not only the article preview for both sites ? 

https://www.heise.de/rss/heise.rdf

https://www.heise.de/security/rss/news-atom.xml

**Comment by martinrotter** [comment]
This is basically the same thing as https://github.com/martinrotter/rssguard/discussions/553.

You can use exact same article filter with those feeds or any other feeds, article filters are universal.

So head to #553, here is proof that same article filter as from #553 works.

![image](https://user-images.githubusercontent.com/1255302/146323055-13f28783-ef22-47a0-a4bf-5b1a0e68560e.png)
